master/getting-started/kubernetes/installation/hosted/canal/canal-etcd.yaml

```diff
@@ -1,515 +1,4 @@
 ---
 layout: null
 ---
-# Canal Version {{site.data.versions[page.version].first.title}}
-# {{site.url}}/{{page.version}}/releases#{{site.data.versions[page.version].first.title}}
-# This manifest includes the following component versions:
-#   calico/node:{{site.data.versions[page.version].first.title}}
-#   calico/cni:{{site.data.versions[page.version].first.components["calico/cni"].version}}
-#   calico/kube-controllers:{{site.data.versions[page.version].first.components["calico/kube-controllers"].version}}
-#   coreos/flannel:v0.9.1
-
-# This ConfigMap can be used to configure a self-hosted Canal installation.
-kind: ConfigMap
-apiVersion: v1
-metadata:
-  name: canal-config
-  namespace: kube-system
-data:
-  # Configure this with the location of your etcd cluster.
-  etcd_endpoints: "https://127.0.0.1:2379"
```

Comment:
interestingly, looks like canal etcd used `https` instead of `http`. I wouldn't have thought that `https` would work...